### PR TITLE
logging middleware

### PR DESCRIPTION
**Pull request by mgranados**

### CODE REVIEW

File: api/routers/requestLogs/list.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+  handler: async function (ctx) {
+    var requestLogs = await RequestLog.dataTables({
+      limit: ctx.request.query.limit || 20,
+      skip: ctx.request.query.start,
+      find: {},
+      sort: '-createdAt'
+    })
+
+    ctx.body = {
+      ...requestLogs
```
Comment (luislizarraga) [inline api/routers/requestLogs/list.js]: No se necesita de-construir el objeto, solo basta ponerlo como va:

`ctx.body = requestLogs`

File: test/api/requestLogs.js [hunk 2]
```diff
@@ -0,0 +1,54 @@
+/* global describe, beforeEach, it */
+require('co-mocha')
+
+const { expect, assert } = require('chai')
+const http = require('http')
+const { clearDatabase, createRequestLog } = require('../utils')
```
Comment (luislizarraga) [inline test/api/requestLogs.js]: No usas `createRequestLog`, no se debería importar :100: :ok_man:

File: api/index.js [hunk 3]
```diff
@@ -21,6 +23,7 @@ app.use(convert(bodyParser()))
 
 app.use(errorHandler)
 app.use(getRequestData)
+app.use(saveRequestLog)
```
Comment (Siedrix) [inline api/index.js]: This middleware its not for all the server, it should be added to selected routers or routes.

File: test/fixtures/requestLogFixture.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+module.exports = {
```
Comment (Siedrix) [inline test/fixtures/requestLogFixture.js]: Change fixtures of real use cases

**Pull request merged**